activities/procurement/principles.md

```diff
@@ -14,3 +14,12 @@ This means the services and products we use need to be:
 * sustainable for the organization as well as society and the planet
 
 The principles of our procurement build on and extend our [cultural values](../../organization/cultural-values.md) and [communication principles](../communication/communication-principles.md).
+
+## Procurement process
+
+To make sure the procurement process is fair, we adhere to the following requirements:
+
+* we request 3 - 4 quotes from different providers
+* check if prices are competitive and in line with the market
+* if products are sustainable and / or local
+* final decision taken with operations team
```

Comment:
I would replace this with something like, 'the order is placed by the operations team and signed off by the chief executive and board of directors if needed'.

Reply:
This was meant as a comment on the last point only